.github/workflows/fork-test.yaml

```diff
@@ -50,7 +50,7 @@ jobs:
             VERSION=${{ matrix.version }}
           context: podtato-services/${{ matrix.component }}/.
           file: podtato-services/${{ matrix.component }}/docker/Dockerfile
-          platforms: linux/amd64
+          platforms: linux/amd64,linux/arm/v7,linux/arm64
```

Comment:
Since `fork-test.yaml` is meant only to build images for its own tests I think it should only build for x86, since the tests will also run only on x86. Building for ARM would require additional time and resources but wouldn't provide benefit in this workflow since those builds won't be used in tests.
If you agree can you please revert this file? Thanks!

Reply:
Fix should be fixed now by setting up Docker QEMU and Buildx for GHA